src/ruby/install.sh

```diff
@@ -210,19 +270,46 @@ else
             DEFAULT_GEMS=""
         fi
     fi
+}
+
+install_previous_version() {
+    repo_url=$(get_github_api_repo_url "$ruby_url")
+    get_previous_version "${ruby_url}" "${repo_url}" RUBY_VERSION
+    set_rvm_install_args
```

Comment:
I don't think we need to re-call this? as it doesn't use VERSION

Hence, let's not isolate to a function and keep code intact.

Reply:
actually, i have now isolated the `original_ruby_version` if given as "latest" from the `ruby_version` found after `find_version_from_git_tags()` fn, so, in `install_previous_version`, it first checks whether user wanted to install the "latest" version, if yes, then it would install the prev. version if source binary for `ruby_version` for latest requested was not found. so, we do need to add it as a separate fn entity so that it can be called for two different versions got initially from `find_version_from_git_tags` and then later from `get_previous_version`

Reply:
`set_rvm_install_args` only checks if the VERSION is `none` or not. In both cases, latest and prev-of-latest, the VERSION will never be none? 

So we can remove set_rvm_install_args() and keep the flow as is?

Reply:
https://github.com/devcontainers/features/blob/main/src/ruby/install.sh#L218

it is used here, and both the times, its the different version for ruby_version variable and then attempting the above as well, so that's why tried to isolate it so can prevent code reuse. 
But if any other way, let me know. 👍🏻